### PR TITLE
Trident 16024 onboarding checklist for cse

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request (PR) adds an onboarding checklist for Cloud SIEM.

Issue number: 
[TRIDENT-16024](https://jasklabs.atlassian.net/browse/TRIDENT-16024)

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/cse/onboarding-checklist-cse.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+---
+id: onboarding-checklist-cse
+title: Onboarding checklist for CSE
```
Comment (kimsauce) [inline docs/cse/onboarding-checklist-cse.md:3]: Titles get titlecased (H2 and below is sentence case)

```suggestion
title: Onboarding Checklist for CSE
```

File: docs/cse/onboarding-checklist-cse.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+keywords:
+  - cloud siem
+  - cse
+  - onboarding
+  - getting started
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+This article provides a high-level checklist of onboarding tasks for Cloud SIEM Enterprise (CSE) administrators. This setup guide ensures that you follow best practices, and helps you obtain maximum value from the CSE by showing you how to leverage all of the features of the Cloud SIEM solution.
```
Comment (kimsauce) [inline docs/cse/onboarding-checklist-cse.md:15]:
```suggestion
This article provides a high-level checklist of onboarding tasks and best practices for Cloud SIEM Enterprise (CSE) administrators. This setup guide shows you how to maximize value from CSE by leveraging all of its features. 
```

File: docs/cse/onboarding-checklist-cse.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+
+## Provision CSE
+
+Sumo Logic must enable (provision) Cloud SIEM Enterprise before you can access it.
+
+* [Create and Manage Organizations (Service Providers)](/docs/manage/manage-subscription/create-manage-orgs-service-providers/). Once Sumo Logic [provisions CSE](/docs/manage/manage-subscription/create-manage-orgs-service-providers#about-cse-provisioning), you see a link labeled **Cloud SIEM Enterprise** on the left side of the Sumo Logic navigation list. <br/><img src={useBaseUrl('img/cse/cse-option-in-left-nav.png')} alt="Cloud SIEM Enterprise menu option" width="300"/>
+* [Manage Organizational Settings](/docs/manage/manage-subscription/manage-org-settings) (optional). If you have multiple Sumo Logic accounts, you may find it useful to [set up a custom subdomain](/docs/manage/manage-subscription/manage-org-settings#set-up-a-customsubdomain) for each of your accounts. Custom subdomains help ensure that requests authenticate to the right account. 
+* [Enable a Support Acccount](/docs/manage/security/enable-support-account/). Enable Support Access if you would like [Sumo Logic Support](https://support.sumologic.com) to access your environment for assistance.
```
Comment (kimsauce) [inline docs/cse/onboarding-checklist-cse.md:28]:
```suggestion
* [Enable a Support Account](/docs/manage/security/enable-support-account/). If you need [Sumo Logic Support](https://support.sumologic.com) assistance, enable this feature to grant our agents permission to access your environment.
```

File: docs/cse/onboarding-checklist-cse.md (hunk 1 above)
Comment (jpipkin1) [reply docs/cse/onboarding-checklist-cse.md]: Fixed by commit "[Updates from review by Kim Pohas](https://github.com/SumoLogic/sumologic-documentation/pull/2009/commits/fd35c19fcc5fb01c9cb0309e3f6d83aeb2dd3d30)".

File: docs/cse/onboarding-checklist-cse.md (hunk 2 above)
Comment (jpipkin1) [reply docs/cse/onboarding-checklist-cse.md]: Fixed by commit "[Updates from review by Kim Pohas](https://github.com/SumoLogic/sumologic-documentation/pull/2009/commits/fd35c19fcc5fb01c9cb0309e3f6d83aeb2dd3d30)".

**Comment by jpipkin1** [comment]
@pkazmir-sumo, could you review this PR for the new CSE onboarding checklist?

**Review by kimsauce — APPROVED** [review]

**Comment by jpipkin1** [comment]
I need to rework this PR per input from Chas Clawson.

**Comment by jpipkin1** [comment]
I made this PR a draft because I'm reworking the content based on a review by Chas Clawson. Once I finish reworking it, I can click "Ready for review" at the bottom of the PR.

For info about draft PRs, see GitHub's article:
[Changing the stage of a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request)

**Comment by jpipkin1** [comment]
I removed the link to the Insight Trainer article (/docs/cse/rules/insight-trainer) because that article won't be published until Friday, April 28 with [PR 2129](https://github.com/SumoLogic/sumologic-documentation/pull/2129). I left out the link for now because it broke the build-and-deploy check.

**Comment by jpipkin1** [comment]
Chas Clawson gave his approval via Slack.

**Pull request merged**